tutorial/source/gplvm.ipynb

```diff
@@ -315,63 +319,67 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now comes the most interesting part. Remember that we have set labels of cells according to its stage: $1$, $2$, $4$, $8$, $16$, $32$, and $64$ (capture time). We will use this information to set priors for the first dimension of $X$, with the hope that after inference, the first dimension of $X$ will still be able to mimic the captured time, and the second dimension can represent the cell branching types (TE, ICM, PE, EPI) at 32-cell stages and 64-cell stages."
+    "Now comes the most interesting part. We know that the observed data $y$ has latent structure: in particular different datapoints correspond to different cell stages. We would like our GPLVM to learn this structure in an unsupervised manner. In principle, if we do a good job of inference then we should be able to discover this structure---at least if we choose reasonable priors. First, we have to choose the dimension of our latent space $X$. We choose $dim(X)=2$, since we would like our model to disentangle 'capture time' ($1$, $2$, $4$, $8$, $16$, $32$, and $64$) from cell branching types (TE, ICM, PE, EPI). Next, when we set the mean of our prior over $X$, we set the first dimension to be equal to the observed capture time. This will help the GPLVM discover the structure we are interested in and will make it more likely that that structure will be axis-aligned in a way that is easier for us to interpret."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 15,
    "metadata": {},
    "outputs": [],
    "source": [
     "captured_time = y.new_tensor([int(cell_name.split(\" \")[0]) for cell_name in df.index.values])\n",
-    "# we scale the time info into the interval [0, 1]\n",
-    "time = (captured_time.log2() - 1) / 5\n",
+    "# we scale the time into the interval [0, 1]\n",
+    "time = captured_time.log2() / 6\n",
```

Comment:
i changed this to get it into the unit interval

Reply:
you are right! I can't believe that I made that mistake. Thanks!